### PR TITLE
Add some super rad utils

### PR DESCRIPTION
**Pull request by childish-sambino**
Will be used laterz

### CODE REVIEW

File: src/services/javascript-utilities.js [hunk 1]
```diff
@@ -5,6 +5,30 @@ const doesObjectHaveProperty = (obj, propertyName) => {
   return Object.prototype.hasOwnProperty.call(obj, propertyName);
 };
 
+const translateKeys = (obj, keyFunc) => {
```
Comment (thinkingserious) [inline src/services/javascript-utilities.js]: Could you add a quick comment describing the utility? I'm not sure what `keyFunc` would be.

File: test/services/naming-conventions.test.js [hunk 2]
```diff
@@ -1,104 +1,108 @@
-const { kebabCase, camelCase, capitalize } = require('../../src/services/naming-conventions');
+const { kebabCase, camelCase, snakeCase, capitalize } = require('../../src/services/naming-conventions');
```
Comment (thinkingserious) [inline test/services/naming-conventions.test.js]: Nice!

File: src/services/javascript-utilities.js (hunk 1 above)
Comment (thinkingserious) [reply src/services/javascript-utilities.js]: Something like: "Translates a complex object's keys to uppercase." If we expect to have more such key translators, then maybe the function should allow for another parameter to specify the translation.

Comment (childish-sambino) [reply src/services/javascript-utilities.js]: Doc added. This function doesn't define _how_ the keys of the object will be translated/converted. It just provides the mechanism to perform the conversion. `keyFunc` is the implementation of how to translate the keys. It accepts the current key and returns the translated version.

**Pull request merged**